benchmarks/benchmarks.py

```diff
@@ -1,42 +1,29 @@
 # Write the benchmarking functions here.
 # See "Writing benchmarks" in the asv docs for more information.
-import subprocess
-import time
 
-from tiled.client import from_uri
+import numpy
 
-HOST = "0.0.0.0"
-PORT = 9040
+from tiled.adapters.array import ArrayAdapter
+from tiled.adapters.mapping import MapAdapter
+from tiled.client import Context, from_context
+from tiled.server.app import build_app
 
 
 class TimeSuite:
-    """
-    An example benchmark that times the performance of various kinds
-    of iterating over dictionaries in Python.
-    """
-
     def setup(self):
-        self.server_process = subprocess.Popen(
```

Comment:
Notice that instead of starting a server in a subprocess, this PR runs the server and client in the same process. Messages are passed directly via Python function calls, the same as it works in unit tests. This takes networking out of the equation and will give me stable results, focused on actual Tiled performance.